server.js

```diff
@@ -18,6 +18,9 @@ const app = next({ dev });
 
 const handle = routes.getRequestHandler(app);
 
+const hasLangCookie = req => req.cookies && req.cookies[LANG_COOKIE_NAME];
```

Comment:
is it possible that `req.cookies` doesn't exist?

Reply:
hm, seems it is not.